app/views/carriers/_carrier_form.html.erb

```diff
@@ -44,6 +44,11 @@
     <%= form.number_field :default_loan_length_days, class: "form-control" %> days
   </div>
 
+  <div class="form-group">
+    <%= form.label :status, 'Status' %>
+    <%= form.select :status, ['Disabled', 'Sold', 'Unavailable' ], {}, {class: "form-control"} %>
```

Comment:
It would be good to have these in an enum, not in the view layer. Check out Rails enums: https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html

Reply:
After adding an enum, it would be good to derive the list of statuses from that active record enum.